workers/data_refinery_workers/processors/utils.py

```diff
@@ -16,20 +13,21 @@
 
 def start_job(kwargs: Dict):
     """Record in the database that this job is being started and
-    retrieves the job's batch from the database and
-    adds it to the dictionary passed in with the key 'batch'."""
+    retrieves the job's batches from the database and
+    adds them to the dictionary passed in with the key 'batches'."""
     job = kwargs["job"]
     job.worker_id = "For now there's only one. For now..."
     job.start_time = timezone.now()
     job.save()
 
-    try:
-        batch = Batch.objects.get(id=job.batch_id)
-    except Batch.DoesNotExist:
-        logger.error("Cannot find batch record with ID %d.", job.batch_id)
+    batch_relations = ProcessorJobsToBatches.objects.filter(processor_job_id=job.id)
+    batches = list(map(lambda x: x.batch, batch_relations))
```

Comment:
This line can be simpler as described above:

`batches = [br.batch for br in batch_relations]`